src/api/java/appeng/api/features/IInscriberRegistry.java

```diff
@@ -12,14 +13,15 @@
  * Lets you manipulate Inscriber Recipes, by adding or editing existing ones.
  *
  * @author thatsIch
- * @version rv2
+ * @version rv3
  * @since rv2
  */
 public interface IInscriberRegistry
 {
 	/**
-	 * Current list of registered recipes, you can modify this if you want too.
-	 * Will never contain a null recipe
+	 * An immutable copy of currently registered recipes.
+	 *
+	 * Use the provided methods to actually modify the inscriber recipes.
```

Comment:
you should notice, that this is quote expensive, since upon each call an immutable copy will be created.

also link the "provided" methods with something like `@see´